MobileFilter/sections/specific_web.txt

```diff
@@ -54,6 +54,9 @@ avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotak
 !
 knowyourmeme.com##hr[data-label="Advertisement"]
 knowyourmeme.com##article:not([data-title]) > .content-unit-refresh-wrapper
+boldsky.com###outbrain_nano
+boldsky.com##iframe[src*="infinite-scroll-ad-mobi.html"]
+boldsky.com##li:not(data-wow-delay)[class="clearfix"]
```

Comment:
It seems the bad rule:
![image](https://user-images.githubusercontent.com/8361299/92409957-28f1d500-f14b-11ea-8a55-9431397526aa.png)